requirements/base.txt

```diff
@@ -189,7 +189,8 @@ lxml==4.6.3
     # via
     #   -r requirements/base.in
     #   xmlsec
-maykin-django-two-factor-auth[phonenumbers]==2.0.2
+#maykin-django-two-factor-auth[phonenumbers]==2.0.2
+git+https://github.com/maykinmedia/django-two-factor-auth/@task-334-fix-login-page-style#egg=django-two-factor-auth
```

Comment:
Never update `base.txt` directly, you should be using the `pip-compile` tooling. See the header of this file:

```
#
# This file is autogenerated by pip-compile
# To update, run:
#
#    ./bin/compile_dependencies.sh
#
```

Reply:
done